Configurations/10-main.conf

```diff
@@ -934,28 +934,28 @@ sub vms_info {
     # in order to build "universal" binary and allow OpenSSL take
     # advantage of NEON when it's available.
     #
-    "android-armeabi" => {
-        inherit_from     => [ "android", asm("armv4_asm") ],
+    "android-armeabi-base" => {
+        inherit_from     => [ asm("armv4_asm") ],
```

Comment:
> when you look at say android-mips, you have to unravel one less template.

I meant "you *would* have to unravel one less template [if there was no android-mips-base]".